impl/impl.go

```diff
@@ -194,7 +194,7 @@ func (m *manager) SendVoucher(ctx context.Context, channelID datatransfer.Channe
 	if channelID.Initiator != m.peerID {
 		return errors.New("cannot send voucher for request we did not initiate")
 	}
-	updateRequest, err := message.VoucherRequest(channelID.ID, voucher.Type(), voucher)
+	updateRequest, err := message1_1.VoucherRequest(channelID.ID, voucher.Type(), voucher)
```

Comment:
one thought about all these renames:

it might make sense, in the root /message directory to alias all of these functions (with a variable) i.e in
`message/message.go`:
```
import (
   "github.com/filecoin-project/go-data-transfer/message/message1_1"
)

var VoucherRequest = message1_1.VoucherRequest
var NewResponse = message1_1.NewResponse
var NewRequest = message1_1.NewRequest
...
```

That way if there is a message1_2 at some point we won't have to make another change like this.


 -- that way, we can avoid another change of this source

Reply:
This is done.